code/game/objects/effects/landmarks/corpsespawner.dm

```diff
@@ -104,7 +104,8 @@
 		M.update_burst()
 		//buckle to nest
 		var/obj/structure/bed/nest/victim_nest = locate() in get_turf(src)
-		victim_nest?.do_buckle(M, silent = TRUE)
+		if(victim_nest)
+			victim_nest.buckle_mob(M, silent = TRUE)
```

Comment:
Why this change, your normally against this type of thing?

Reply:
Sounds nuts, but the profiling results show it, `var?.var` is more efficient than `var && var.var`, but `var?.proc()` is actually less than `var && var.proc()`